----- Begin Fatal Exception 24-Jan-2013 12:08:59 CET-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named eleTreeProd_L1Study_with_opts.py
Exception Message:
python encountered the error: <type 'exceptions.IOError'>
[Errno 2] No such file or directory: 'eleTreeProd_L1Study_with_opts.py'
----- End Fatal Exception -------------------------------------------------
